internal/services/cdn/cdn_frontdoor_profile_resource.go

```diff
@@ -187,7 +187,7 @@ func resourceCdnFrontDoorProfileDelete(d *pluginsdk.ResourceData, meta interface
 	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorProfileID(d.Id())
+	id, err := parse.FrontDoorProfileIDInsensitively(d.Id())
```

Comment:
this is coming from the Resource ID and so should be being parsed case-sensitively:

```suggestion
	id, err := parse.FrontDoorProfileID(d.Id())
```

Reply:
Fixed.